src/helpers/index.js

```diff
@@ -22,6 +22,10 @@ export const getItemType = (item) => {
   return ITEM_TYPES.TASK
 }
 
+export const getItemId = (item) => {
+  return item.wp_guid ? item.wp_guid : item.id.toString()
```

Comment:
Jos haluut hienostella niin tän pitäis toimia myös näin: `item.wp_guid || item.id.toString()`